docs/section5/Rule5-46.md

```diff
@@ -0,0 +1,40 @@
+
+# Envelope - Rule 5-46  
+
+**Rule ID:** 5-46  
+**Rule Description:** The infiltration shall be modeled using the same methodology and adjustments for weather and building operation in both the proposed design and the baseline building design.  
+**Rule Assertion:** For all zones, B-RMR infiltration:modeling_method_name = P-RMR infiltration:modeling_method_name and B-RMR infiltration:modeling_method = P-RMR infiltration:modeling_method.  
+**Appendix G Section:** Section G3.1-5(b) Building Envelope Modeling Requirements for the Proposed design and Baseline  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:**  None  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:** 
+
+  1. match_data_element()
+
+## Rule Logic:  
+
+- For each building segment in the Proposed model: `for building_segment_p in P_RMR.building.building_segments:`  
+
+  - For each thermal block in building segment: `for thermal_block_p in building_segment_p.thermal_blocks:`  
+
+    - For each zone in thermal block: `for zone_p in thermal_block_p.zones:`
+
```

Comment:
Thermal block element has been removed from schema. I think we should start using the abbreviated path in the RDSs to avoid confusion.